hiero-enterprise-base/pom.xml

```diff
@@ -42,6 +42,12 @@
       <artifactId>junit-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+  <groupId>org.mockito</groupId>
+  <artifactId>mockito-core</artifactId>
+  <version> 5.11.0</version>
```

Comment:
We define the versions for all dependencies in the root pom. Please remove the version at this place and add it to the root pom.xml